src/include/msvc.h

```diff
@@ -114,23 +114,7 @@ WT_PAUSE(void)
 static inline void
 WT_READ_BARRIER(void)
```

Comment:
We have
```
#ifndef _M_AMD64
#error "Only x64 is supported with MSVC"
#endif
```
up on line 10, but it'd be useful to have comments in `WT_READ_BARRIER` and `WT_WRITE_BARRIER` that say they're x86 specific.

Reply:
[244c742](https://github.com/wiredtiger/wiredtiger/pull/10028/commits/244c742f639e500284b01cde5ac377f6ade3901f)